### DISCUSSION TITLE
DateTimeFormats.ofFormat("02/30/2024") is not ambiguous

### DISCUSSION THREAD

**Discussion by xingyutangyuan** [google/mug#58] [Ideas]
The document calls out `01/02/2023` as ambiguous. But `02/30/2024` or `30/02/2024` aren't ambiguous.

If these are supported, then Europe formats can mostly be supported without having to use the `<placeholder>` syntax.

**Comment by fluentfuture** [comment]
Well. I can't think of good reasons against it. Letting people to use natural datetime as is is a selling point.

Let me play around and see how much work this needs.
